tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing
+
+This page describes how to contribute to [APIView](../../../src//dotnet/APIView/APIViewWeb/CONTRIBUTING.md) language level parsers.
+Specifically how to create or update a language parser to produce tree style tokens for APIView.
+
+## Creating Tree Style Tokens
```

Comment:
This document is hard to follow. APIView v1 was poorly documented (not document?) and now we have v1 and v2 mixed together. This is the best time to ensure that this new version is properly documented. There are two issues here that should be addressed:
1. Documenting v2. "How should someone who is only working with v2 use it?"  This is what v1 was lacking.
2. Conversion guide from v1 to v2. "How do I convert my existing parse to this new model?" This won't be as useful long-term since there are a finite number of parsers. If the baseline documentation was solid, it could also be very short. "1. Create the following new types. 2. Update this existing model. 3. Etc...".

Especially given that the tokens are totally different between v1 and v2, it's going to be nigh impossible for parser authors who are steeped in v1 to effectively convert their parsers.

Reply:
For what it is worth, I've been porting the Java parser to the new v2 structure for the last few weeks. It is largey different in terms of structure, but I did reuse a large amount of the existing Java v1 code. I am happy to act as a sounding board for people in the process of conversion - because I'm on the other side now. It isn't trivial.